src/app/dataset-view/additional-components/metadata-component/components/form-components/schema-field/schema-field.component.html

```diff
@@ -10,34 +11,33 @@
             (cdkDropListDropped)="drag($event)"
         >
             <ng-container matColumnDef="name">
-                <th mat-header-cell *matHeaderCellDef width="40%">name</th>
+                <th mat-header-cell *matHeaderCellDef>name</th>
```

Comment:
For some reason, if I type a name, and press ENTER, this UI is inserting new row, although I still have a type to select:

![image](https://user-images.githubusercontent.com/8875909/230764419-b0aebbab-a2f5-40f4-a243-16e5197d5e92.png)

Reply:
Fixed